projects/default/controllers/ros/RosCamera.cpp

```diff
@@ -76,9 +76,15 @@ RosCamera::~RosCamera() {
 // creates a publisher for camera image with
 // a [4 x ImageWidth x ImageHeight] {unsigned char} array
 ros::Publisher RosCamera::createPublisher() {
+  createCameraInfoPublisher("camera_info");
```

Comment:
This parameter is always set to "camera_info", therefore I believe it should be hard-coded in the `createCameraInfoPublisher` method:
```suggestion
  createCameraInfoPublisher();
```

Reply:
Hi, I can adept this to a fixed name and update the pull request.

Reply:
That would be great.

Reply:
Changes should be done in the newest Push. Feel free to confirm, if I did everything correct. ^^